README.md

```diff
@@ -119,9 +119,6 @@ We provide some instructions to guide you through the preparation for the follow
     ```bash
     git clone https://github.com/streamnative/charts.git
     ```
-    ```bash
```

Comment:
after cloning the repo, it will create a directory `charts`. So you need to enter the chart dir first, no?

Reply:
No, because the `scripts` directory does not exist in `charts/charts`.